recipes/signac/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "signac" %}
+{% set version = "0.6.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_rev: v{{ version }}
+  git_url: https://bitbucket.org/glotzer/signac
```

Comment:
Things are a bit backed up because of issue ( https://github.com/conda-forge/staged-recipes/issues/2093 ). We could pull that recipe temporarily to allow things to clear.